tests/test_relion_interop.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+import numpy as np
+import pytest
+
+from aspire.source import RelionSource, Simulation
+from aspire.volume import Volume
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
+
+
+STARFILE = ["rln_proj_65.star", "rln_proj_64.star"]
+
+
+@pytest.fixture(params=STARFILE, ids=lambda x: f"resolution={x}", scope="module")
+def starfile(request):
+    filepath = request.param
+    return os.path.join(DATA_DIR, filepath)
+
+
+@pytest.fixture(scope="module")
+def sources(starfile):
+    rln_src = RelionSource(starfile)
+
+    # Generate Volume used for Relion projections.
+    # Note, `downsample` is a no-op for resolution 65.
+    vol_path = os.path.join(DATA_DIR, "clean70SRibosome_vol.npy")
+    vol = Volume(np.load(vol_path), dtype=rln_src.dtype).downsample(rln_src.L)
+
+    # Create Simulation source using Volume and angles from Relion projections.
+    # Note, for odd resolution Relion projections are shifted by 1 pixel in x and y.
+    offsets = 0
+    if rln_src.L % 2 == 1:
+        offsets = -np.ones((rln_src.n, 2), dtype=rln_src.dtype)
+
+    sim_src = Simulation(
```

Comment:
random `amplitues`?... 

I realize it might not matter for the FRC based test. However, if this test is viewed as a recipe for interop we should probably try to get those details correct.

Can you remind me how close in absolute and relative terms this real space projection comparison was?  If too difficult to codify into a tight unit test, perhaps it would still make a decent gallery.

Reply:
Set amplitudes to 1.

After normalization of both image sets, we're off by a little more than 2% using the frobenius norm.

I'll put together a tutorial comparing the image sets.

Reply:
Added a tutorial to the gallery.